examples/bind.amp.html

```diff
@@ -0,0 +1,129 @@
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+  <script async custom-element="amp-bind" src="https://cdn.ampproject.org/v0/amp-bind-0.1.js"></script>
+  <script async custom-element="amp-carousel" src="https://cdn.ampproject.org/v0/amp-carousel-0.1.js"></script>
+  <style amp-custom>
+    body {
+      font-family: Verdana;
+      background-color: black;
+    }
+    h2 {
+      display: block;
+      margin: 20px;
+      font-weight: normal;
+    }
+
+    h2.title {
+      font-size: 16pt;
+      color: #ff1744;
+    }
+
+    h2.price {
+      font-size: 13pt;
+      color: #9C27B0;
+      font-style: italic;
+    }
+
+    .thumbnails amp-img {
+      opacity: 0.5;
+    }
+
+    .thumbnails amp-img[selected] {
+      opacity: 1;
+    }
+
+    .thumbnails {
+      display: flex;
+      justify-content: space-between;
+    }
+
+    .thumbnails amp-img {
+      flex: 1;
+    }
+  </style>
+</head>
+<body>
+  <h2 class="title">Guitar AMP</h2>
+  <h2 class="price">$500</h2>
+  <div>
+    <amp-carousel slide=0 id="carousel-1" width=1 height=1 layout=responsive type=slides controls>
+      <amp-bind-attribute layout="nodisplay"
+          attr="slide"
+          value="{{selectedSlide}}">
+      </amp-bind-attribute>
+
+      <amp-bind-event layout="nodisplay"
+          on="slidechange"
+          variable="selectedSlide"
+          value="{{ampEventData}}">
+      </amp-bind-event>
+
+      <amp-img src="http://i.ebayimg.com/images/g/h~kAAOSwyQtVytYY/s-l500.jpg" width=1 height=1 layout=responsive></amp-img>
+      <amp-img src="http://i.ebayimg.com/images/g/TnEAAOSwPcVVytYc/s-l1600.jpg" width=1 height=1 layout=responsive></amp-img>
+      <amp-img src="http://i.ebayimg.com/images/g/ZRsAAOSwgQ9VytYh/s-l1600.jpg" width=1 height=1 layout=responsive></amp-img>
+      <amp-img src="http://i.ebayimg.com/images/g/6WUAAOSwHnFVytYk/s-l1600.jpg" width=1 height=1 layout=responsive></amp-img>
+    </amp-carousel>
+
+    <div class="thumbnails">
+      <amp-img selected src="http://i.ebayimg.com/images/g/h~kAAOSwyQtVytYY/s-l500.jpg" width="1" height="1"
+        layout="responsive">
+        <amp-bind-attribute layout="nodisplay"
+            attr="selected"
+            toggle="{{selectedSlide === 0}}">
+        </amp-bind-attribute>
+        <amp-bind-event layout="nodisplay"
+            on="tap"
+            variable="selectedSlide"
```

Comment:
I think the carousel should automatically declare it's selected slide as a var that can be used as is. E.g.
```
<amp-bind-attribute
         attr="selected"
         toggle="{{carousel-1.slide === 0}}">
<amp-bind-event
    on="tap"
    variable="carousel-1.slide"
    value="{{0}}"
  >
```

Using a 3-rd variable may also be a good idea is more complicated cases. But in the simple cases, it'd be good to reference the AMP element's public vars directly.

That being said, the simplest case when we bind and update the same variable - we can even make things simpler by creating an "actionable variable":

```
<amp-bind
    attribute="selected"
    target="carousel-1.slide"
    value="0"
  >
```

I haven't come up with better names yet, but functionally it could be very simple. We'd basically say that the attribute "selected" is bound to variable "carousel-1.slide" with value "0". That connection becomes a two-way connection. When slide changes, the attribute is re-evaluated; when attribute changes, the carousel's slide is re-evaluated.

Anyhow, just some musings...